dev-python/patch-ng/patch-ng-1.17.2.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+inherit distutils-r1
+
+DESCRIPTION="Library to parse and apply unified diffs, fork of dev-python/patch"
+HOMEPAGE="https://github.com/conan-io/python-patch-ng"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
```

Comment:
I see Github has tests enabled, could you implement those in this ebuild?

Reply:
In Github there no 1.17.2 tagged release, only on pypi, with no tests. Will add them in future releases.